### ISSUE TITLE
Fail build on asciidoctor warnings

### ISSUE THREAD

**Issue by gAmUssA** [asciidoctor/asciidoctor-gradle-plugin#255]
Hello,

How I can tell gradle to fail build if there are asciidoctor warnings?
Thank you

**Comment by mojavelinux** [comment]
I think this should be part of #240. All warnings are now routed through the logger and the log handler is now pluggable in AsciidoctorJ. It's just a matter of consulting the messages that were recorded and deciding when to fail the build.

See https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/363/files.

**Comment by AlexanderZobkov** [comment]
Yes, at the moment, this information is hard to find :smile:.  
I think it should be in https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/154

**Comment by ysb33r** [comment]
Correct. The answer lies within #154. 

This has been implemented in 2.0-alpha.3  See syntax https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/154#issuecomment-405521147. 

For 1.5.x there is a largish workaround that can be hidden inside a separate Gradle include file. See https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/154#issuecomment-234213913

**Comment by ysb33r** [comment]
The feature is fully supported in 2.0.0. You can do something like

```groovy
asciidoctorj {
  fatalWarnings ~/fail if warning matches the pattern/
  fatalWarnings missingIncludes()
}
```
